dart/lib/src/attachment_extensions/_io_attachment_extensions.dart

```diff
@@ -0,0 +1,40 @@
+import 'dart:io';
+
+import '../attachment.dart';
+import '../scope.dart';
+
+extension IoScopeExtensions on Scope {
```

Comment:
`Scope` only offers `addAttachment(attachment)`

the `Attachment` has enough factories for it, right?

Reply:
This is tricky. On web it doesn't make sense to have the `fromPath` and `fromFile` methods (and it's actually not available). So I put them in extension methods to make them available depending on if its web or not. Though it doesn't work quite right yet.

Reply:
the behaviour is correct, what I mean is, the extension should be in the Attachment class, not in the Scope